centos_bridge_network_configuration.sh

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env bash
+
+set -xe
+
+# shellcheck disable=SC1091
+source lib/logging.sh
+# shellcheck disable=SC1091
+source lib/common.sh
+# shellcheck disable=SC1091
+source lib/network.sh
+
+if [ "$MANAGE_PRO_BRIDGE" == "y" ]; then
+    # Create a veth iterface peer.
+    sudo ip link add ironicendpoint type veth peer name ironic-peer
+    # Adding an IP address in the libvirt definition for this network results in
+    # dnsmasq being run, we don't want that as we have our own dnsmasq, so set
+    # the IP address here
+    if [ ! -e /etc/sysconfig/network-scripts/ifcfg-provisioning ] ; then
+        if [[ "${PROVISIONING_IPV6}" == "true" ]]; then
+          echo -e "DEVICE=provisioning\nTYPE=Bridge\nONBOOT=yes\nNM_CONTROLLED=no\nIPV6_AUTOCONF=no\nIPV6INIT=yes" | sudo dd of=/etc/sysconfig/network-scripts/ifcfg-provisioning
```

Comment:
I'm wondering if IPV6INIT should be there. SInce this is now only used as a bridge, we probably don't care about layer 3 configuration, so it might need to be no ?

Reply:
As discussed, I have added a condition to run httpd-infra container with PROVISIONING_INTERFACE var.